src/components/partials/IdentitySearch/IdentitySearch.vue

```diff
@@ -29,6 +29,8 @@
                         :options="chainOptions"
                         class="select"
                         v-model:selected-key="selectedChainKey"
+                        :hasInPrefix="true"
```

Comment:
@sosan-neikbeen HTML attributes should be in kebap-case instead of camelCase --> ```has-in-prefix```